+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Mon Oct 16 10:47:09 2023 - INFO  - Start to prepare the test environment.
+ OLD_LANG=C.UTF-8
+ export LANG=en_US.UTF-8
+ LANG=en_US.UTF-8
+ DNF_INSTALL iputils
+ pkgs=iputils
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs iputils --node 1 --tempfile ''
Mon Oct 16 10:48:03 2023 - INFO  - pkgs:(iputils) is already installed
+ tmpfile2=
+ '[' -z '' ']'
+ tmpfile=
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Mon Oct 16 10:48:04 2023 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Mon Oct 16 10:48:04 2023 - INFO  - Start to run test.
+ ping6 -a fe80::71c4:b4e8:c0d4:bfed -c 3
+ grep '3 received'
+ CHECK_RESULT 1 0 0 'ping6 -a execute failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='ping6 -a execute failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'ping6 -a execute failed'
+ LOG_ERROR 'ping6 -a execute failed'
+ message='ping6 -a execute failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'ping6 -a execute failed'
Mon Oct 16 10:48:08 2023 - ERROR - ping6 -a execute failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iputils_ping6_01.sh line 33'
+ message='oe_test_iputils_ping6_01.sh line 33'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iputils_ping6_01.sh line 33'
Mon Oct 16 10:48:08 2023 - ERROR - oe_test_iputils_ping6_01.sh line 33
+ '[' 0 -eq 1 ']'
+ return 0
+ ping6 -A fe80::71c4:b4e8:c0d4:bfed -c 3
+ grep '3 received'
+ CHECK_RESULT 1 0 0 'ping6 -A execute failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='ping6 -A execute failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'ping6 -A execute failed'
+ LOG_ERROR 'ping6 -A execute failed'
+ message='ping6 -A execute failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'ping6 -A execute failed'
Mon Oct 16 10:48:12 2023 - ERROR - ping6 -A execute failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iputils_ping6_01.sh line 35'
+ message='oe_test_iputils_ping6_01.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iputils_ping6_01.sh line 35'
Mon Oct 16 10:48:12 2023 - ERROR - oe_test_iputils_ping6_01.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ ping6 -B fe80::71c4:b4e8:c0d4:bfed -c 3
+ grep '3 received'
+ CHECK_RESULT 1 0 0 'ping6 -B execute failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='ping6 -B execute failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'ping6 -B execute failed'
+ LOG_ERROR 'ping6 -B execute failed'
+ message='ping6 -B execute failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'ping6 -B execute failed'
Mon Oct 16 10:48:16 2023 - ERROR - ping6 -B execute failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iputils_ping6_01.sh line 37'
+ message='oe_test_iputils_ping6_01.sh line 37'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iputils_ping6_01.sh line 37'
Mon Oct 16 10:48:16 2023 - ERROR - oe_test_iputils_ping6_01.sh line 37
+ '[' 0 -eq 1 ']'
+ return 0
+ ping6 fe80::71c4:b4e8:c0d4:bfed -c 3
+ grep '3 received'
+ CHECK_RESULT 1 0 0 'ping6 -c execute failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='ping6 -c execute failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'ping6 -c execute failed'
+ LOG_ERROR 'ping6 -c execute failed'
+ message='ping6 -c execute failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'ping6 -c execute failed'
Mon Oct 16 10:48:20 2023 - ERROR - ping6 -c execute failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iputils_ping6_01.sh line 39'
+ message='oe_test_iputils_ping6_01.sh line 39'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iputils_ping6_01.sh line 39'
Mon Oct 16 10:48:20 2023 - ERROR - oe_test_iputils_ping6_01.sh line 39
+ '[' 0 -eq 1 ']'
+ return 0
+ ping6 -D fe80::71c4:b4e8:c0d4:bfed -c 3
+ grep '3 received'
+ CHECK_RESULT 1 0 0 'ping6 -D execute failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='ping6 -D execute failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'ping6 -D execute failed'
+ LOG_ERROR 'ping6 -D execute failed'
+ message='ping6 -D execute failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'ping6 -D execute failed'
Mon Oct 16 10:48:24 2023 - ERROR - ping6 -D execute failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iputils_ping6_01.sh line 41'
+ message='oe_test_iputils_ping6_01.sh line 41'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iputils_ping6_01.sh line 41'
Mon Oct 16 10:48:24 2023 - ERROR - oe_test_iputils_ping6_01.sh line 41
+ '[' 0 -eq 1 ']'
+ return 0
+ ping6 -d fe80::71c4:b4e8:c0d4:bfed -c 3
+ grep '3 received'
+ CHECK_RESULT 1 0 0 'ping6 -d execute failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='ping6 -d execute failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'ping6 -d execute failed'
+ LOG_ERROR 'ping6 -d execute failed'
+ message='ping6 -d execute failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'ping6 -d execute failed'
Mon Oct 16 10:48:27 2023 - ERROR - ping6 -d execute failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iputils_ping6_01.sh line 43'
+ message='oe_test_iputils_ping6_01.sh line 43'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iputils_ping6_01.sh line 43'
Mon Oct 16 10:48:28 2023 - ERROR - oe_test_iputils_ping6_01.sh line 43
+ '[' 0 -eq 1 ']'
+ return 0
+ dmidecode -s system-product-name
+ grep Virtual
+ ping6 -f fe80::71c4:b4e8:c0d4:bfed -c 3
+ grep '3 received'
+ CHECK_RESULT 1 0 0 'ping6 -f execute failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='ping6 -f execute failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'ping6 -f execute failed'
+ LOG_ERROR 'ping6 -f execute failed'
+ message='ping6 -f execute failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'ping6 -f execute failed'
Mon Oct 16 10:48:31 2023 - ERROR - ping6 -f execute failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iputils_ping6_01.sh line 51'
+ message='oe_test_iputils_ping6_01.sh line 51'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iputils_ping6_01.sh line 51'
Mon Oct 16 10:48:32 2023 - ERROR - oe_test_iputils_ping6_01.sh line 51
+ '[' 0 -eq 1 ']'
+ return 0
+ ping6 -h
+ grep Usage
Usage
+ CHECK_RESULT 0 0 0 'ping6 -h execute failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='ping6 -h execute failed'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Mon Oct 16 10:48:32 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 7
+ test 7 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Mon Oct 16 10:48:33 2023 - ERROR - failed to execute the case.
+ exit 7
+ post_test
+ LOG_INFO 'start to restore the test environment.'
+ message='start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'start to restore the test environment.'
Mon Oct 16 10:48:33 2023 - INFO  - start to restore the test environment.
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z '' ]]
+ [[ -z '' ]]
+ LOG_WARN 'no thing to do.'
+ message='no thing to do.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level warn --message 'no thing to do.'
Mon Oct 16 10:48:33 2023 - WARN  - no thing to do.
+ return 0
+ export LANG=C.UTF-8
+ LANG=C.UTF-8
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Mon Oct 16 10:48:34 2023 - INFO  - End to restore the test environment.
